src/ImageSequencer.js

```diff
@@ -197,6 +197,25 @@ ImageSequencer = function ImageSequencer(options) {
     else modulesdata = modules[name][1];
     return modulesdata;
   }
+
+  function toString(step_name,options){
+    if(step_name){
+      return `${step_name}(${Object.keys(options).map(key=>key + ':' + options[key]).join(',')})`
+    }
+    else{
+      return  copy(this.images.image1.steps).map(step=>{
+        let inputs = modulesInfo(step.options.name).inputs;
+        
+        if(!inputs)
+          inputs = {}
+        
+        for(let input in inputs){
+          inputs[input] = step.options[input];
+        }
+        return toString(step.options.name,inputs);
+      }).join(',');
```

Comment:
Yes, i think it's preferable to define the `function forEachStep(step) {` function separately then use it, because by this time, we're not sure what's being joined without searching up in the source -- just for readability!

Also, i'm sorry, i put a lot of my comments into the `dist` version. My bad! But I think you can still follow.